src/internal/type.ts

```diff
@@ -62,3 +65,47 @@ export interface IncompleteUploadedBucketItem {
   uploadId: string
   size: number
 }
+
+export interface ItemBucketMetadataList {
+  Items: MetadataItem[]
+}
+
+export interface ItemBucketMetadata {
+  [key: string]: any
```

Comment:
There seems to be the following lint warning here:
```
Unexpected any. Specify a different type
```

Reply:
do you suggest disable this lint rule or use a more specific type？

Reply:
I think we can use something like `// eslint-disable-next-line @typescript-eslint/no-explicit-any` for both cases here